resources/assets/less/bem/beatmapset-score-big.less

```diff
@@ -0,0 +1,174 @@
+/**
+ *    Copyright 2015-2016 ppy Pty. Ltd.
+ *
+ *    This file is part of osu!web. osu!web is distributed with the hope of
+ *    attracting more community contributions to the core ecosystem of osu!.
+ *
+ *    osu!web is free software: you can redistribute it and/or modify
+ *    it under the terms of the Affero GNU General Public License version 3
+ *    as published by the Free Software Foundation.
+ *
+ *    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+ *    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *    See the GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+.beatmapset-score-big {
+  display: flex;
+  flex-direction: column;
+
+  height: 90px;
+  width: calc(~"100% - 6px");
+
+  .default-border-radius;
+  .default-box-shadow;
+
+  margin: @spacing * 2 auto;
+
+  &--first-score {
+    border: 3px solid @yellow;
```

Comment:
I'd rather use `box-shadow: 0 0 0 3px @yellow, 0 4px @box-shadow-spread @box-shadow-color;` than this and all that 1.7s.